twitter-server/src/com/treode/twitter/finagle/http/package.scala

```diff
@@ -125,14 +173,20 @@ package object http {
         TxClock.parse (value) .getOrThrow (new BadRequestException (s"Bad time for $name: $value"))
       }
 
+	//Maintaining compatibility with un-updated servers that use older headers
```

Comment:
This is just a temporary change. Since I was just seeing the finagle and movies servers, I just assumed that the other servers' tests might still be using the old headers. Either I can change this or Sanjay will anyway change these with his changes for the other 2 servers. And about the name change, I first used different functions, but then thought that the server files might be more meaningful if we can indicate the actual condition in the variable name. Also, when using different functions, I had to define 2 conditionTxClock functions since the default values for GET and PUT requests were different. So I just left them with the same names. Is this okay or shall I rename the functions?